Mathlib/MeasureTheory/Measure/FiniteMeasureProd.lean

```diff
@@ -89,7 +89,7 @@ lemma map_prod_map {α' : Type*} [MeasurableSpace α'] {β' : Type*} [Measurable
     (μ.map f).prod (ν.map g) = (μ.prod ν).map (Prod.map f g) := by
   apply Subtype.ext
   simp only [val_eq_toMeasure, toMeasure_prod, toMeasure_map]
-  rw [Measure.map_prod_map _ _ f_mble g_mble] <;> infer_instance
+  rw [Measure.map_prod_map _ _ f_mble g_mble]
```

Comment:
I can only assume this is somehow due to https://github.com/leanprover/std4/pull/407?

Reply:
@digama0, does this make sense to you?

Reply:
this generates a linter warning on master

Reply:
I was curious to find out how this happened, and it seems that the LTAR file corresponding to this file was produced by the run https://github.com/leanprover-community/mathlib4/actions/runs/7105988780/job/19344435622 , which failed, but we commit the results anyway for files that built. Subsequent builds therefore skip this file, and so the noisy files lint does not fire.

Reply:
Wow, that sounds like a pretty nasty problem with built-in linters +our CI setup